### ISSUE TITLE
Normal Way of setting training=False for batch Normalization finetuning

### ISSUE THREAD

**Issue by zulkaif121** [keras-team/keras-io#653]
The way we set training=False in baseModel while finetuning converts our baseModel in a functional object which can be seen in model summary that the baseModel appears like Xception(functional) instead it is good idea to iterate through whole model and set batch norm layers.trainable =False that way the model stays in layers form and we can apply gradCam technique.

**Comment by sachinprasadhs** [comment]
Hi, 

Thanks for reporting the issue.

Could you please elaborate more on the issue.

If you are referring to any examples or tutorials, could you please provide the link for it.

For latest examples, refer https://keras.io/examples/

**Comment by zulkaif121** [comment]
I am out of country right now. But what I mean is that when we require to
use gradcam technique which uses a specific layer if the model is
functional type we dont have any way to access its layers. Using 2nd method
does not convert model into functional module. It can be checked by
printing model. Its layers can be accessed and that way we can use gradcam.
I will go back home in 15 days and then will send the example of what i
meant.

**Comment by sachinprasadhs** [comment]
Hi,

The tutorial which you are referring to has been updated to Keras 3, could you please verify and close the issue. Thanks!

https://keras.io/examples/vision/grad_cam/
